packages/EasyErrorHandler/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -91,12 +93,22 @@
         ->arg('$defaultIsVerbose', param(BridgeConstantsInterface::PARAM_IS_VERBOSE));
 
     // Error codes provider
-    $services->set(ErrorCodesProviderInterface::class, ErrorCodesProvider::class)
+    $services->set('error_codes_provider.by_interface', ErrorCodesProvider::class)
```

Comment:
Can we rename `ErrorCodesProvider` to `ErrorCodesByInterfaceProvider` without BC?

Reply:
Renamed

Reply:
Done

Reply:
Thanks